doc/11-svg-images.md

```diff
@@ -0,0 +1,10 @@
+# Custom Hypersync App Icons and Branding
+
+Every custom Hyperysnc app must include four SVG files which the user will see while working with the Hyperysnc app in the Hyperproof UI. These files must be `.SVG` files, and they must follow certain naming and size rules.
+
+| File              | Description                                                                                       | Sizing      |
+| ----------------- | ------------------------------------------------------------------------------------------------- | ----------- |
+| `icon-small.svg`  | Small icon as shown when viewing the list of proof generated by the Hypersync.                    | 20px x 20px |
+| `icon-medium.svg` | Medium icon. Shown in the existing connection tiles in the Choose App modal.                      | 32px x 32px |
+| `icon-large.svg`  | Large icon as shown when viewing the Hypersync cards on the Automation tab of controls or labels. | 48px x 48px |
+| `brand.svg`       | Brand image shown when creating a new Hypersync or editing an existing Hypersync.                 | 32px high   |
```

Comment:
Do we have any constraints on the width of the brand image?

Reply:
Not currently.  I'm sure you could come up with something that was 3000 pixels wide and that wouldn't look very good, but we don't have a hard constraint.